code/src/components/AnimatedPhrase.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import './AnimatedPhrase.css'; // Import CSS file
+
+const words = ['Front-end', 'Back-end', 'AI', 'UX/UI'];
+
+const AnimatedPhrase = () => {
+  const [currentWordIndex, setCurrentWordIndex] = useState(0);
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      setCurrentWordIndex((currentWordIndex + 1) % words.length);
+    }, 2500);
+    return () => clearInterval(intervalId);
+  }, [currentWordIndex]);
+
+  const phraseWithWord = words[currentWordIndex];
+
+  return (
+    <h1 className="compleate-phrase">
```

Comment:
Could this code have been shortened  by just using the text "Find your perfect  {phrasewithword} mentor" in a paragraph? 🤔